### PR TITLE
Add lambda prebuild option to CLI env vars, readd docker build make target

### PR DESCRIPTION
**Pull request by dfangl**
## Motivation
We are currently missing the `LAMBDA_PREBUILD_IMAGES` environment variable in the CLI env var flags, which is added in this commit.
I also ordered the LAMBDA section of the list alphabetically.

In addtion to this, similar as in -ext, I readded the `make docker-build` target, so developers can build the image easily again, without having to look up the variables necessary for the `docker-helper.sh`.

## Changes
* Add `LAMBDA_PREBUILD_IMAGES` to CI env vars
* Readd `docker-build` make target

### CODE REVIEW

**Review by alexrashed — APPROVED** [review]
LGTM! Thanks for also adding the make target, I thought we could maybe get rid of it, but for the sake of our devx it's good to re-introduce it 💯

File: localstack-core/localstack/config.py [hunk 1]
```diff
@@ -1230,7 +1222,16 @@ def use_custom_dns():
     "LAMBDA_LIMITS_CREATE_FUNCTION_REQUEST_SIZE",
     "LAMBDA_LIMITS_MAX_FUNCTION_ENVVAR_SIZE_BYTES",
     "LAMBDA_LIMITS_MAX_FUNCTION_PAYLOAD_SIZE_BYTES",
+    "LAMBDA_PREBUILD_IMAGES",
```
Comment (joe4dev) [inline localstack-core/localstack/config.py:1225]: 👍

File: Makefile [hunk 2]
```diff
@@ -130,6 +137,9 @@ format-modified:          ## Run ruff to format only modified code
 init-precommit:    		  ## install te pre-commit hook into your local git repository
 	($(VENV_RUN); pre-commit install)
 
+docker-build:
```
Comment (joe4dev) [inline Makefile:140]: That fails for me on macOS M1 (haven't looked into it):

```
Dockerfile:36
--------------------
  35 |     # Install runtime OS package dependencies
  36 | >>> RUN --mount=type=cache,target=/var/cache/apt \
  37 | >>>     apt-get update && \
  38 | >>>         # Install dependencies to add additional repos
  39 | >>>         apt-get install -y --no-install-recommends \
  40 | >>>             # Runtime packages (groff-base is necessary for AWS CLI help)
  41 | >>>             ca-certificates curl gnupg git make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps iproute2 xz-utils libatomic1
  42 |
--------------------
ERROR: failed to solve: process "/bin/sh -c apt-get update &&         apt-get install -y --no-install-recommends             ca-certificates curl gnupg git make openssl tar pixz zip unzip groff-base iputils-ping nss-passwords procps iproute2 xz-utils libatomic1" did not complete successfully: exit code: 100
make: *** [docker-build] Error 1
```

**Review by joe4dev — APPROVED** [review]
Thanks for adding the missing `LAMBDA_PREBUILD_IMAGES` option 👍

File: Makefile (hunk 2 above)
Comment (joe4dev) [reply Makefile:140]: my Docker VM was running out of space hitting this issue: https://github.com/docker/for-mac/issues/6963

`docker image prune` recovered ~80GB and fixed the build 👍

**Pull request merged**